Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -916,7 +916,7 @@ static bool refersToAllAspects(WGPUTextureFormat format, WGPUTextureAspect aspec
     } break;
     case WGPUTextureDimension_2D:
     case WGPUTextureDimension_3D: {
-        auto checkedBlockSizeTimesTextureDimension = checkedProduct<uint32_t>(blockSize * m_device->limits().maxTextureDimension2D);
+        auto checkedBlockSizeTimesTextureDimension = checkedProduct<uint32_t>(blockSize, m_device->limits().maxTextureDimension2D);
```

Comment:
`blockSize` is at most `16` and maxTextureDimension2D is limited to `16384`, so it was impossible to overflow but better not to have the typo here